docs/source/using_pytorch.rst

```diff
@@ -21,21 +21,63 @@ Analog layers
-
-* :class:`~aihwkit.nn.modules.conv.AnalogConv2d`:
-  applies a 2D convolution over an input signal composed of several input
-  planes. It is the counterpart of PyTorch `nn.Conv2d`_ layer.
-
-* :class:`~aihwkit.nn.modules.conv.AnalogConv3d`:
-  applies a 3D convolution over an input signal composed of several input
-  planes. It is the counterpart of PyTorch `nn.Conv3d`_ layer.
+Convolution layers
+~~~~~~~~~~~~~~~~~~
+
++--------------------------------------------------------------+-----------------------------------------------------+---------------------+
+| Analog Layer                                                 | Description                                         | PyTorch Counterpart |
++==============================================================+=====================================================+=====================+
+| :class:`~aihwkit.nn.modules.linear.AnalogLinear`             | | Applies a linear transformation to the input data | `nn.Linear`_        |
+|                                                              | | (layers) and Functions.                           |                     |
++--------------------------------------------------------------+-----------------------------------------------------+---------------------+
+| :class:`~aihwkit.nn.modules.conv.AnalogConv1d`               | | Applies a 1D convolution over an input signal     | `nn.Conv1d`_        |
+|                                                              | | composed of several input planes.                 |                     |
++--------------------------------------------------------------+-----------------------------------------------------+---------------------+
+| :class:`~aihwkit.nn.modules.conv.AnalogConv2d`               | | Applies a 2D convolution over an input signal     | `nn.Conv2d`_        |
+|                                                              | | composed of several input planes                  |                     |
++--------------------------------------------------------------+-----------------------------------------------------+---------------------+
+| :class:`~aihwkit.nn.modules.conv.AnalogConv3d`               | | Applies a 3D convolution over an input signal     | `nn.Conv3d`_        |
+|                                                              | | composed of several input planes                  |                     |
++--------------------------------------------------------------+-----------------------------------------------------+---------------------+
+| :class:`~aihwkit.nn.modules.linear_mapped.AnalogLinearMapped`| | Similar to AnalogLinearMapped but constrains the  | `nn.Linear`_        |
+|                                                              | | the maximal in and/or out dimension of an analog  |                     |
+|                                                              | | tile.                                             |                     |
```

Comment:
could we add here something like:
and will construct multiple tiles (as many as necessary to cover the weight matrix). Splitting, concatenation, and partial sum addition is done in digital.